packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -175,3 +179,17 @@ const prepareJsonForCmd = (json: any) => {
   }
   return jsonString;
 };
+
+// There is a chance multiple documents will be returned consecutively, without proper containment. Look for them and parse as an array
+const splitUnsafeDocuments = (documentResponse: string): any[] => {
+  const DOCUMENT_SEPARATOR = "}\n{";
```

Comment:
This could incorrectly be triggered by a multi-line field value that contains this string

Reply:
Here's what the start/end of a document looks like
```
	"yextRoutableCoordinate": {
		"latitude": 33.20214104581977,
		"longitude": -117.36811858975076
	}
}              <----- need to find this point
{
	"__": {
		"name": "robots_txt",
		"staticPage": {
			"plugin": {}
		},
		"templateType": "JS"
	},
	"_site": {
```
Because the `__` property is default, we can check for that, and since a field starting with three underscores is technically possible (though not created by the user via custom field). I should be able to check for just the opening of the underscores `"}\n{\n  "__`. This would greatly reduce the already small chance for an incorrect match, basically down to someone storing a stream response with the alternateLanguageFields feature in a multi-line text field.

Reply:
It seems a somewhat odd that the data returned by invoking `yext sites generate-test-data [other flags]` would not include commas to separate multiple objects for a proper json format. Instead of having workarounds here trying to split the document by some specific regex and reconstruct the json object, is it possible to update `yext sites generate-test-data` to add commas between documents? -- I assume in `gocode/src/yext/cli/sites/generatetestdata/generatetestdata.go` somewhere, it's appending the documents together to send in one go

Reply:
@yen-tt some further discussion on slack

Reply:
So the backend will provide as a json array instead of the missing comma situation here. Definitely make this easier without worrying about edge cases. Let's update this pr after that work is done.

Reply:
good idea!